main_test.go

```diff
@@ -17,6 +13,7 @@ func TestGetHelpText(t *testing.T) {
 	}
 }
 
+/*
```

Comment:
sneaky sneaky

Reply:
Oh yeah, sorry about that. I meant to add some more tests :)